lib/calendar.ts

```diff
@@ -71,17 +72,17 @@ interface CalendarImpl {
   dateFromFields(
     fields: Params['dateFromFields'][0],
     options: NonNullable<Params['dateFromFields'][1]>,
-    calendar: Temporal.Calendar
+    calendar: string
```

Comment:
I agree, but I'm loath to introduce a deviation from upstream for something as minor as parameter names, esp. because in TS we know the type from the annotation. I'd prefer to save this until after upstream commits have quieted down. 

Sorry to be so sensitive about this but I've spent so much time on rebasing recently!